src/OperationResponse.php

```diff
@@ -495,4 +514,10 @@ private function hasProtoResponse()
     {
         return !is_null($this->lastProtoResponse);
     }
+
+    private function isNewSurfaceOperationsClient(): bool
+    {
+        return !$this->operationsClient instanceof LegacyOperationsClient
+            && false !== strpos(get_class($this->operationsClient), '\\Client\\');
```

Comment:
Personally, I would flip this condition so that it reads better instead of being a double negative "not false".
```suggestion
            && true === strpos(get_class($this->operationsClient), '\\Client\\');
```

Altogether it confirms "this client is not instance of Legacy client and has 'Client' namespce"

Reply:
makes sense to me, good call

Reply:
Oh, I forgot that we actually have a very good reason for doing this 😛 

The `strpos` function returns `false` to mean the needle string was not found in the haystack string. So we HAVE to use `false !==`. We could potentially do `is_int(strpos(get_class($this->operationsClient), '\\Client\\');` instead, but I'm not sure that's more readable...

Reply:
Ah ok that makes sense. Good to leave it as is. Thanks!